NitroxClient/GameLogic/Bases/EntityUtils/NitroxModule.cs

```diff
@@ -0,0 +1,45 @@
+using NitroxClient.MonoBehaviours;
+using NitroxClient.Unity.Helper;
+using NitroxModel.DataStructures.GameLogic.Entities.Bases;
+using NitroxModel_Subnautica.DataStructures;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.Bases.EntityUtils;
+
+public static class NitroxModule
```

Comment:
The main issue here is some of the code contained in those static classes runs on Unity so we can't just move it to NitroxModel. Also we can't move the Entities to NitroxClient because we need their stuff for NitroxServer...